session/session_test.go

```diff
@@ -3231,6 +3231,29 @@ func (s *testSessionSuite2) TestPerStmtTaskID(c *C) {
 	c.Assert(taskID1 != taskID2, IsTrue)
 }
 
+func (s *testSessionSuite) TestDoDDLJobQuit(c *C) {
+	// test https://github.com/pingcap/tidb/issues/18714, imitate DM's use environment
+	store, err := mockstore.NewMockStore(mockstore.WithStoreType(mockstore.MockTiKV))
+	c.Assert(err, IsNil)
+	session.SetSchemaLease(10 * time.Millisecond)
+	dom, err := session.BootstrapSession(store)
```

Comment:
It is done in `SetUpSuite`, could we remove these code?

Reply:
because we'll manually close `dom`, so can't use suite-shared variable

Reply:
Got it.